packages/arb-token-bridge-ui/src/util/fastBridges.ts

```diff
@@ -118,6 +124,10 @@ export function getFastBridges({
     [FastBridgeNames.Synapse]: {
       imageSrc: Synapse,
       href: getBridgeDeepLink(FastBridgeNames.Synapse)
+    },
+    [FastBridgeNames.deBridge]: {
```

Comment:
the list is in alphabetical order so i think we should move it to after `Connext`